series-advisor/src/components/Form.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+const Form = () => (
+	<form className="login-container__form">
+		<h1 className="login-container__form--title">Zaloguj się</h1>
+		<input className="login-container__form--input" type="text" placeholder="E-mail..." required />
```

Comment:
There's no label for both email and password input. Don't use `placeholder` as a label, it's typically for example values.